msteams-platform/bots/how-to/Teams conversational AI/assistants-api-quick-start.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Assistants API quick start guide
+author: surbhigupta
+description: In this module, learn how to quickly try the Assistants API.
+ms.topic: conceptual
+ms.localizationpriority: high
+zone_pivot_groups: assistant-ai-library-quick-start
+ms.author: v-bvishnu
+ms.date: 05/20/2024
+---
+
+# Assistants API quick start guide
```

Comment:
Should we update the title as "Quick start guide for using Assistants API with Teams AI library" or similar, as Assistants API is an openai / azure product, currently it sounds like it's a quick start guide for another product.

Reply:
Updated.